src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -227,14 +228,19 @@ class adminLogs extends viewModelBase {
         return item.includes("EXCEPTION:") || item.includes("Exception:") || item.includes("FATAL ERROR:");
     }
     
+    private isCustomizedItem(item: string): boolean {
+        return (item.includes("Connection established") || item.includes("Connection closed") || item.includes("Connection paused"));
```

Comment:
remove ( )

Reply:
done

Reply:
also maybe we should have more bullet proof way of checking that. Maybe create function to append studio messages and format will be like:
2020-12-21T11:05:45.12345678Z, -, Information, Studio, HERE GOES message.

Then we can matching using: `-, Information, Studio, ` string.

Reply:
done